test/built-ins/RegExp/CharacterClassEscapes/character-class-digit-class-escape-plus-quantifier.js

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2018 Leo Balter.  All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: prod-CharacterClassEscape
+description: >
+    Compare range for Digit class escape, \\d+ with flags 
+info: |
+    This is a generated test, please checkout https://github.com/bocoup/test262-regexp-generator
+    for any changes.
+
+    CharacterClassEscape[U] ::
+        d
+        D
+        s
+        S
+        w
+        W
+
+    21.2.2.12 CharacterClassEscape
+    
+    The production CharacterClassEscape :: d evaluates as follows:
+        Return the ten-element set of characters containing the characters 0 through 9 inclusive.
+    The production CharacterClassEscape :: D evaluates as follows:
+        Return the set of all characters not included in the set returned by CharacterClassEscape :: d.
+    The production CharacterClassEscape :: s evaluates as follows:
+        Return the set of characters containing the characters that are on the right-hand side of
+        the WhiteSpace or LineTerminator productions.
+    The production CharacterClassEscape :: S evaluates as follows:
+        Return the set of all characters not included in the set returned by CharacterClassEscape :: s.
+    The production CharacterClassEscape :: w evaluates as follows:
+        Return the set of all characters returned by WordCharacters().
+    The production CharacterClassEscape :: W evaluates as follows:
+        Return the set of all characters not included in the set returned by CharacterClassEscape :: w.
+---*/
+
+var re = /\d+/;
+var matchingRange = /[0-9]+/;
+var msg = '"\\u{REPLACE}" should be in range for \\d+ with flags ';
+
+var i;
+var fromEscape, fromRange, str;
+for (i = 0; i < 0xFFFF; i++) {
+
+    str = String.fromCharCode(i);
+    fromEscape = !str.replace(re, 'test262');
+    fromRange = !str.replace(re, 'test262');
+    assert.sameValue(fromEscape, fromRange, msg.replace('REPLACE', i));
+
+
+    str += str;
+    fromEscape = !str.replace(re, 'test262');
+    fromRange = !str.replace(re, 'test262');
+    assert.sameValue(fromEscape, fromRange, msg.replace('REPLACE', String(i) + i));
```

Comment:
This results in display such as `\u{123123}` for `i = 0x7B`. That’s confusing… Maybe add a helper like 

```js
toHex(0x7B);
// → '\\u{7B}'
```

You could then just `.repeat(2)` its output in this particular case.